test/soca/gw/prep_com.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+set -e
+
+project_binary_dir=$1
+project_source_dir=$2
+
+# Stage history and restart files following the "COM" structure
+
+COM=${project_binary_dir}/test/soca/gw/COM/gdas.20180415
+
+mkdir -p ${COM}/06/model_data/ice/history
```

Comment:
since this is in g-w, can we use the templates/bash function already available? or is that overkill?

Reply:
no, def. not overkill if the functionality exists somewhere else.